profile/src/main/java/com/scottlogic/deg/profile/reader/atomic/ConstraintReaderHelpers.java

```diff
@@ -63,6 +64,10 @@ public static OffsetDateTime parseDate(String value) {
     }
 
     public static Types getFieldType(String type) {
+        if (type == null) {
+            return null;
```

Comment:
Why would we ever want this to be null? Shouldn't this just fall through the switch statement and throw an exception?

Reply:
nulls null pointer switch statements.

this is a bit of laziness, the null is handled later and throws a validation exception which says which field was untyped.

but i didn't think it was worth handling 'properly' here as near future work should make this pointless.